spec/features/create_invoice_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
```

Comment:
@yunixon please move all this specs to `spec/features/invoices/` directory and name like `create_spec.rb`, `index_spec.rb` e.t.c